src/components/collect-qr.vue

```diff
@@ -59,11 +59,7 @@ const imgHtml = ref('');
 const margin = 15;
 const textFontSize = 16;
 
-// The QR code is rendered to a canvas (possibly with additional icons and text)
-// and then the canvas data is converted to an image via toDataURL and used as the source
-// of an image tag.
-// It is assumed that props will not change and the QR image will only be rendered once.
-onMounted(() => {
+const renderQrCode = () => {
```

Comment:
Realized that the QR code wasn't changing when the managed/legacy settings changed, oops! 

It actually doesn't change on staging or the QA server either.

Reply:
Issue noted here: https://github.com/getodk/central/issues/885